edgelet/edgelet-http-workload/src/server/mod.rs

```diff
@@ -91,3 +95,42 @@ impl NewService for WorkloadService {
         future::ok(self.clone())
     }
 }
+
+fn get_key_handle(
```

Comment:
`fn get_derived_identity_key_handle` perhaps, since now there's also `get_derived_enc_key_handle` and to be consistent with its naming scheme.

Reply:
Fixed